flink-end-to-end-tests/test-scripts/common_kubernetes.sh

```diff
@@ -48,10 +48,19 @@ function setup_kubernetes_for_linux {
       sudo rm "$(which minikube)"
     fi
 
+    # Retry loop download minikube.
+    local MINIKUBE_DOWNLOAD_RETRY_COUNT=2
+    for i in $(seq 1 ${MINIKUBE_DOWNLOAD_RETRY_COUNT}); do
```

Comment:
I mentioned it in [the other PR](https://github.com/apache/flink/pull/23528#issuecomment-1768147345) already, but just for the sake of transparency: You could use retry_times (see [apache/flink:flink-end-to-end-tests/test-scripts/common.sh:883ff](https://github.com/apache/flink/blob/d8630cb5db0608a630de95df0dd1d0c9f0b56aa2/flink-end-to-end-tests/test-scripts/common.sh#L883)) instead of reimplementing the logic.

Would that work?